lib/ally.rb

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2011 Instructure, Inc.
+#
+# This file is part of Canvas.
+#
+# Canvas is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, version 3 of the License.
+#
+# Canvas is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+require 'oauth'
+
+module Ally
+
+  class Client
+
+    ##
+    # Creates a new Ally client
+    def initialize(client_id, secret, base_url="prod.ally.ac")
```

Comment:
Oversight, it should've included the protocol. I've removed the default argument as that really should be provided.